code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -121,7 +124,8 @@
 		if(3)
 			adjustBruteLoss(50)
 
-
+/mob/living/simple_animal/hostile/megafauna/proc/SetRecoveryTime(var/buffer_time)
```

Comment:
No ```var/```

This keeps happening so often lately I'm really tempted to write a script to auto-fail travis.